app/Http/Controllers/Auth/LoginController.php

```diff
@@ -118,6 +118,7 @@ function ($vatsimUser) {
                 $user->save();
 
                 $user->syncRatings($vatsimUser->rating->id, $vatsimUser->pilot_rating->rating);
+                $user->updatePrimaryMembership($vatsimUser->division->code, $vatsimUser->region->code);
```

Comment:
Do you want to update every time someone logs in? Surely just check if it has changed?

Reply:
It does check if it changes in the method's logic.